lib/sparql.jison

```diff
@@ -567,7 +567,7 @@ SubSelect
     ;
 SelectClauseItem
     : VAR -> toVar($1)
-    | '(' (Expression | ConstTriple) 'AS' VAR ')' -> expression($2, { variable: toVar($4) })
+    | '(' (Expression | VarTriple) 'AS' VAR ')' -> expression($2, { variable: toVar($4) })
```

Comment:
We'll need an `ensureSparqlStar` around some of that (fc204e9f57564b56d7054fc79b005319ecc58c52); perhaps need to split the rule (but avoid if possible, that's more expensive).

Reply:
Should we need to call `ensureSparqlStar` here in addition to `VarTriple` / `ConstTriple` rule? I think it will fail when matching these rules first.

Reply:
Yeah; this is why we might want to split this into two rules.